src/Doctrine/ODM/Repository/BaseDocumentRepository.php

```diff
@@ -75,7 +75,7 @@ function () use ($criteria, $orderBy, $limit, $offset) {
                 $results = $this->createSelectQuery($criteria, $orderBy, $limit, $offset)->execute();
 
                 // Return as array of documents (and not the iterator)
-                return $results->toArray();
+                return array_values($results->toArray());
```

Comment:
Might be better to rollback to original
`return $results->toArray(false);`
see

Reply:
No, because `\Doctrine\ODM\MongoDB\Cursor` is deprecated. That's why I changed it in the first place.